package.json

```diff
@@ -15,7 +15,7 @@
     },
     "scripts": {
         "asset:build": "yarn && yarn --cwd ./asset build",
-        "build": "tsc --build",
+        "build": "tsc --build --force",
```

Comment:
```suggestion
        "build": "tsc --build",
```

Reply:
I think you actually want to add the `--force` on the `yarn setup` script, not build.

* https://github.com/terascope/elasticsearch-assets/blob/master/package.json#L31
* https://github.com/terascope/file-assets/blob/master/package.json#L24
* https://github.com/terascope/standard-assets/blob/master/package.json#L21